tests/integration/test_model.py

```diff
@@ -120,6 +120,7 @@
             _deploy_in_loop(new_loop, model_name))
         await model._wait_for_new('application', 'ubuntu')
         assert 'ubuntu' in model.applications
+        await model.disconnect()
```

Comment:
Why was this added?  The `CleanModel` context handler ought to disconnect this for us.  Did you run across an issue with it not being disconnected?